### ISSUE TITLE
MongoDB support

### ISSUE THREAD

**Issue by Anonyfox** [roapi/roapi#175]
It seems that roapi can fetch data _directly_ from supported RDBMS (like postgres) as well as slurping things from JSON files - I wonder if it would be possible to make MongoDB as a direct datasource available? That would be awesome and even unlock running SQL queries over badly designed mongo collections!

**Comment by houqp** [comment]
Yes, it's all doable, someone just need to write some glue code to convert documents from mongodb into the arrow format and we will get all the query capability for free ;)

**Comment by elliot14A** [comment]
@houqp can you give me more context on how to do this?

**Comment by elliot14A** [comment]
hey @houqp can i get more context on this, like should use `mongodb` crate to fetch all the documents first or is there another way around?

**Comment by amotl** [comment]
Hi. MongoDB read support would be sweet, indeed. Could the MongoDB integrations for Apache Arrow help here, or is it too much Python inside?

- https://github.com/mongodb-labs/mongo-arrow
